beacon/validator/src/test/java/tech/pegasys/teku/validator/coordinator/BlockFactoryDenebTest.java

```diff
@@ -72,25 +71,17 @@ void shouldCreateBlockContents() {
   @Test
   void shouldCreateBlindedBlockContentsWhenBlindedBlockRequested() {
 
-    final BlindedBlobsBundle blindedBlobsBundle = prepareBlindedBlobsBundle(spec, 3);
+    final SszList<SszKZGCommitment> blobKzgCommitments = prepareBlobKzgCommitments(spec, 3);
 
     final BlockContainer blockContainer =
         assertBlockCreated(1, spec, false, state -> prepareValidPayload(spec, state), true);
 
     assertThat(blockContainer).isInstanceOf(BlindedBlockContents.class);
     final BlindedBlockContainer blindedBlockContainer = blockContainer.toBlinded().orElseThrow();
     assertThat(blindedBlockContainer.getBlock().getBody().getOptionalBlobKzgCommitments())
-        .hasValueSatisfying(blobKzgCommitments -> assertThat(blobKzgCommitments).hasSize(3));
+        .hasValue(blobKzgCommitments);
     assertThat(blindedBlockContainer.getBlindedBlobSidecars())
-        .hasValueSatisfying(
-            blindedBlobSidecars ->
-                assertThat(blindedBlobSidecars)
-                    .hasSize(3)
-                    .map(BlindedBlobSidecar::getBlobRoot)
-                    .hasSameElementsAs(
-                        blindedBlobsBundle.getBlobRoots().stream()
-                            .map(SszBytes32::get)
-                            .collect(Collectors.toList())));
+        .hasValueSatisfying(blindedBlobSidecars -> assertThat(blindedBlobSidecars).isEmpty());
```

Comment:
I'd rather keep it broken and disabled to not forget we need to make it work than leave with incorrect check

Reply:
good point